content/en/home/datenschutz.md

```diff
@@ -0,0 +1,184 @@
+---
```

Comment:
Die Datei hier brauchen wir nicht :) Die Datenschutzerklärung ist ja schon in /content/en/datenschutz.md drin, also ist das hier doppelt